chaosk8s/actions.py

```diff
@@ -52,11 +53,11 @@ def kill_microservice(name: str, ns: str = "default",
     To work, the deployment must have a `service` label matching the
```

Comment:
Update documentation to include label_selector option?

Reply:
Good catch!